Get rid of allow + -ing (#5045)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDataGrid/1 Configuration/masterDetail/template.md

The master row's key.

##### field(detailInfo.watch): function()

> Allows to track a variable and performing actions when it changes. Applies when [repaintChangesOnly](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/repaintChangesOnly.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#repaintChangesOnly') is **true**.       

Reviewer: ```suggestion
Allows you to track a variable and respond to value changes. Applies when [repaintChangesOnly](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/repaintChangesOnly.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#repaintChangesOnly') is **true**.       
```
